ZUT-PokerPlanningClient/src/app/project-planning/members/members.component.html

```diff
@@ -1 +1,15 @@
 <p>members works!</p>
+
+<!--This is just to present how it works. To be removed later.-->
+<app-game-deck></app-game-deck>
+
+<script>
+    var cards = document.getElementById("game-card-item");
+    for (var i = 0; i < cards.length; ++i) {
+        cards[i].addEventListener("click", function() {
+            var selectedCard = cards.getElementByClassName("active");
```

Comment:
Yes, definitely this should be handled in .ts file.

We can have it for now and then we shall replace it.